Add debug feature module section + require inspector (#1089)
spatialos/gdk-for-unity: docs/modules/debug/overview.md

# Debug Feature Module

> This feature module adds tools to help understand and debug your SpatialOS project.

Reviewer: ```suggestion
This feature module provides tooling to debug your SpatialOS project.
```

just to make it more aligned with the other docs

## Installation

> ### 1. Add the package

Reviewer: ```suggestion
```

there is only one step, no need for enumeration

> 

Reviewer: ```suggestion
```
